CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+<!--
+Copyright 2016 F5 Networks Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Contributing Guide for f5-openstack-ansible
+If you have found this that means you you want to help us out. Thanks in advance for lending a hand! This guide should get you up and running quickly and make it easy for you to contribute.  If we don't answer your questions here and you need help or just want to say hi, shoot us an email at <f5_openstack_ansible@f5.com>.
+
+## Issues
+Creating issues is good, creating good issues is even better. Filing meaningful bug reports with lots of information in them helps us figure out what to fix when and how it impacts our users. We like bugs because it means people are using our code, and we like fixing them even more.
+ 
+## Pull Requests
+If you are submitting a pull request, you need to make sure that you have done a few things first.
+
+* If an issue doesn't exist, file one.
+* Make sure you have tested your code because we are going to do that when when you make your PR.  You don't want _The Hat_ because your request fails unit tests.
+*  Clean up your git history because no one wants to see 75 commits for one issue
+*  Use our [commit template](.git-commit-template.txt)
+*  Use our pull request template
```

Comment:
Create the magic .github directory with the following files:
PULL_REQUEST_TEMPLATE.md
ISSUE_TEMPLATE.md

See the agent repo for samples.